Code/GraphMol/MolStandardize/testCharge.cpp

```diff
@@ -261,6 +261,52 @@ void testChargedAromatics() {
   BOOST_LOG(rdDebugLog) << "Finished" << std::endl;
 }
 
+void testUnchargerProtonationOnly() {
+  BOOST_LOG(rdDebugLog)
+      << "-----------------------\n Testing removal of formal charges limited to "
+         "changing the protonation state (no redox via addition/removal of H-)"
+      << std::endl;
+  {
+    // simple test verifying that for protic compounds the behavior
+    // should be the same as usual
+    auto m1 = "C[N+](C)(C)CC[O-]"_smiles;
+    TEST_ASSERT(m1);
+    // with force=false the zwitterion should stay unmodified
+    MolStandardize::Uncharger uncharger1(false, false, true);
```

Comment:
Here and below: Its a lot easier to understand what these bool options are supposed to be doing if you used named variables for them